config/contentSecurityPolicy.js

```diff
@@ -5,6 +5,8 @@ module.exports = {
       'assets.adobedtm.com',
       '*.demdex.net',
       '*.hotjar.com:*',
+      '*.hotjar.io',
```

Comment:
can you add this change to the changelog too please?

Reply:
Done